src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -17,14 +17,14 @@ import FormField from '../FormField';
 import ProjectIconSelection from '../ProjectIconSelection';
 import TextInput from '../TextInput';
 
-import type { Project } from '../../types';
+import type { Project, Dispatch } from '../../types';
 
 type Props = {
   project: Project | null,
   isVisible: boolean,
   dependenciesChangingForProject: boolean,
-  hideModal: () => void,
-  saveProjectSettings: (string, string, Project) => void,
+  hideModal: Dispatch<typeof actions.hideModal>,
+  saveProjectSettings: Dispatch<typeof actions.saveProjectSettingsStart>,
```

Comment:
I think this pattern is pretty nice and should be generalized for all actions to make sure that when they change in `actions/index.js` they trigger a flow check in the components that use them. cc @joshwcomeau